pandas/tests/arrays/string_/test_string.py

```diff
@@ -230,6 +230,26 @@ def test_reduce(skipna):
     assert result == "abc"
 
 
+@pytest.mark.parametrize("method", ["min", "max"])
+@pytest.mark.parametrize("skipna", [True, False])
+def test_min_max(method, skipna):
+    arr = pd.Series(["a", "b", "c", None], dtype="string")
+    result = getattr(arr, method)(skipna=skipna)
+    if skipna:
+        expected = "a" if method == "min" else "c"
+        assert result == expected
+    else:
+        assert result is pd.NA
+
+
+@pytest.mark.parametrize("method", ["min", "max"])
+def test_min_max_numpy(method):
+    arr = pd.Series(["a", "b", "c", None], dtype="string")
```

Comment:
so we should also test here for the actual array (and not put in a Series), maybe parametrize over pd.array and pd.Series as constructor